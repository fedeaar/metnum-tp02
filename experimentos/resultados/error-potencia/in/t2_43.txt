9.098914000000000000e+06 9.262900000000000000e+04 3.768809000000000000e+06 8.309050000000000000e+05 -1.207172000000000000e+06 -1.398399000000000000e+06 -1.671248000000000000e+06 9.285890000000000000e+05 -1.498967000000000000e+06 2.014465000000000000e+06 5.172520000000000000e+05 -3.003330000000000000e+05 7.112540000000000000e+05 -3.660100000000000000e+04 1.042879000000000000e+06 1.943568000000000000e+06 -1.174459000000000000e+06 -1.407522000000000000e+06 7.776760000000000000e+05 1.423282000000000000e+06
9.262900000000000000e+04 7.676313000000000000e+06 1.567069000000000000e+06 -1.656134000000000000e+06 2.057220000000000000e+06 -7.055530000000000000e+05 -6.381680000000000000e+05 2.369380000000000000e+05 8.829050000000000000e+05 2.045019000000000000e+06 -6.164800000000000000e+05 -1.150896000000000000e+06 -4.754280000000000000e+05 -1.397729000000000000e+06 1.273365000000000000e+06 7.668400000000000000e+04 -1.604792000000000000e+06 1.147037000000000000e+06 -4.457580000000000000e+05 2.898715000000000000e+06
3.768809000000000000e+06 1.567069000000000000e+06 7.378961000000000000e+06 9.298800000000000000e+04 -3.004799000000000000e+06 5.525400000000000000e+05 -4.427610000000000000e+05 -1.526125000000000000e+06 -2.230822000000000000e+06 1.578178000000000000e+06 -1.632550000000000000e+05 -1.381485000000000000e+06 -1.038200000000000000e+04 -9.054900000000000000e+05 1.213203000000000000e+06 8.391090000000000000e+05 -2.459453000000000000e+06 1.556443000000000000e+06 3.273437000000000000e+06 -1.639450000000000000e+05
8.309050000000000000e+05 -1.656134000000000000e+06 9.298800000000000000e+04 6.691518000000000000e+06 -2.571002000000000000e+06 -6.342800000000000000e+04 -1.141436000000000000e+06 6.584130000000000000e+05 -8.109130000000000000e+05 2.562533000000000000e+06 1.041402000000000000e+06 -8.449320000000000000e+05 1.895362000000000000e+06 9.877000000000000000e+05 4.935320000000000000e+05 -1.265179000000000000e+06 -1.716708000000000000e+06 1.000362000000000000e+06 2.785550000000000000e+05 -3.507103000000000000e+06
-1.207172000000000000e+06 2.057220000000000000e+06 -3.004799000000000000e+06 -2.571002000000000000e+06 7.801249000000000000e+06 -2.089941000000000000e+06 -1.137286000000000000e+06 2.525408000000000000e+06 2.829403000000000000e+06 -1.505437000000000000e+06 1.307268000000000000e+06 1.967941000000000000e+06 7.384510000000000000e+05 -1.429972000000000000e+06 -1.210513000000000000e+06 3.433730000000000000e+05 5.301640000000000000e+05 -2.160419000000000000e+06 -2.069102000000000000e+06 1.624883000000000000e+06
-1.398399000000000000e+06 -7.055530000000000000e+05 5.525400000000000000e+05 -6.342800000000000000e+04 -2.089941000000000000e+06 6.629531000000000000e+06 -4.273910000000000000e+05 -1.990384000000000000e+06 -4.908940000000000000e+05 2.192880000000000000e+06 -1.642578000000000000e+06 7.583540000000000000e+05 1.494032000000000000e+06 3.704510000000000000e+05 -1.060603000000000000e+06 1.285301000000000000e+06 2.922550000000000000e+05 -1.949800000000000000e+04 1.019273000000000000e+06 -1.483049000000000000e+06
-1.671248000000000000e+06 -6.381680000000000000e+05 -4.427610000000000000e+05 -1.141436000000000000e+06 -1.137286000000000000e+06 -4.273910000000000000e+05 6.817554000000000000e+06 -4.718900000000000000e+04 -6.874700000000000000e+04 -1.775113000000000000e+06 1.467423000000000000e+06 1.350170000000000000e+05 -5.818160000000000000e+05 -6.433030000000000000e+05 -1.093189000000000000e+06 1.082612000000000000e+06 2.173440000000000000e+06 1.446660000000000000e+05 -5.805070000000000000e+05 1.575534000000000000e+06
9.285890000000000000e+05 2.369380000000000000e+05 -1.526125000000000000e+06 6.584130000000000000e+05 2.525408000000000000e+06 -1.990384000000000000e+06 -4.718900000000000000e+04 7.026645000000000000e+06 2.402473000000000000e+06 -2.267850000000000000e+05 2.373731000000000000e+06 7.592400000000000000e+04 1.256977000000000000e+06 -1.600998000000000000e+06 5.665080000000000000e+05 -2.306660000000000000e+06 4.750830000000000000e+05 -2.559738000000000000e+06 -1.913718000000000000e+06 9.640860000000000000e+05
-1.498967000000000000e+06 8.829050000000000000e+05 -2.230822000000000000e+06 -8.109130000000000000e+05 2.829403000000000000e+06 -4.908940000000000000e+05 -6.874700000000000000e+04 2.402473000000000000e+06 6.733004000000000000e+06 -1.892237000000000000e+06 1.039413000000000000e+06 2.092232000000000000e+06 3.584650000000000000e+05 1.038945000000000000e+06 -1.360627000000000000e+06 -5.698120000000000000e+05 4.716258000000000000e+06 -1.639884000000000000e+06 -4.810200000000000000e+05 9.185230000000000000e+05
2.014465000000000000e+06 2.045019000000000000e+06 1.578178000000000000e+06 2.562533000000000000e+06 -1.505437000000000000e+06 2.192880000000000000e+06 -1.775113000000000000e+06 -2.267850000000000000e+05 -1.892237000000000000e+06 7.172454000000000000e+06 1.712324000000000000e+06 4.636070000000000000e+05 1.173887000000000000e+06 -7.733290000000000000e+05 -2.110694000000000000e+06 3.254066000000000000e+06 -2.453494000000000000e+06 1.006045000000000000e+06 4.334740000000000000e+05 -5.609220000000000000e+05
5.172520000000000000e+05 -6.164800000000000000e+05 -1.632550000000000000e+05 1.041402000000000000e+06 1.307268000000000000e+06 -1.642578000000000000e+06 1.467423000000000000e+06 2.373731000000000000e+06 1.039413000000000000e+06 1.712324000000000000e+06 7.894440000000000000e+06 -4.148600000000000000e+05 1.814528000000000000e+06 -5.626970000000000000e+05 -4.856549000000000000e+06 3.080197000000000000e+06 -5.195530000000000000e+05 -1.690151000000000000e+06 1.079558000000000000e+06 -1.405131000000000000e+06
-3.003330000000000000e+05 -1.150896000000000000e+06 -1.381485000000000000e+06 -8.449320000000000000e+05 1.967941000000000000e+06 7.583540000000000000e+05 1.350170000000000000e+05 7.592400000000000000e+04 2.092232000000000000e+06 4.636070000000000000e+05 -4.148600000000000000e+05 4.674388000000000000e+06 6.411050000000000000e+05 -4.690350000000000000e+05 -1.150656000000000000e+06 7.002960000000000000e+05 3.633523000000000000e+06 -6.148000000000000000e+03 -1.697396000000000000e+06 5.564200000000000000e+05
7.112540000000000000e+05 -4.754280000000000000e+05 -1.038200000000000000e+04 1.895362000000000000e+06 7.384510000000000000e+05 1.494032000000000000e+06 -5.818160000000000000e+05 1.256977000000000000e+06 3.584650000000000000e+05 1.173887000000000000e+06 1.814528000000000000e+06 6.411050000000000000e+05 5.279455000000000000e+06 6.589320000000000000e+05 -3.884750000000000000e+05 3.779080000000000000e+05 -5.820080000000000000e+05 -2.393153000000000000e+06 -1.509820000000000000e+05 -1.858485000000000000e+06
-3.660100000000000000e+04 -1.397729000000000000e+06 -9.054900000000000000e+05 9.877000000000000000e+05 -1.429972000000000000e+06 3.704510000000000000e+05 -6.433030000000000000e+05 -1.600998000000000000e+06 1.038945000000000000e+06 -7.733290000000000000e+05 -5.626970000000000000e+05 -4.690350000000000000e+05 6.589320000000000000e+05 6.432651000000000000e+06 -1.729471000000000000e+06 2.361663000000000000e+06 1.606849000000000000e+06 -4.647130000000000000e+05 1.873400000000000000e+04 -1.022423000000000000e+06
1.042879000000000000e+06 1.273365000000000000e+06 1.213203000000000000e+06 4.935320000000000000e+05 -1.210513000000000000e+06 -1.060603000000000000e+06 -1.093189000000000000e+06 5.665080000000000000e+05 -1.360627000000000000e+06 -2.110694000000000000e+06 -4.856549000000000000e+06 -1.150656000000000000e+06 -3.884750000000000000e+05 -1.729471000000000000e+06 7.937777000000000000e+06 -5.676257000000000000e+06 -1.156108000000000000e+06 1.665451000000000000e+06 1.056871000000000000e+06 1.263718000000000000e+06
1.943568000000000000e+06 7.668400000000000000e+04 8.391090000000000000e+05 -1.265179000000000000e+06 3.433730000000000000e+05 1.285301000000000000e+06 1.082612000000000000e+06 -2.306660000000000000e+06 -5.698120000000000000e+05 3.254066000000000000e+06 3.080197000000000000e+06 7.002960000000000000e+05 3.779080000000000000e+05 2.361663000000000000e+06 -5.676257000000000000e+06 8.636661000000000000e+06 3.214820000000000000e+05 -3.244160000000000000e+05 9.737690000000000000e+05 -1.173720000000000000e+05
-1.174459000000000000e+06 -1.604792000000000000e+06 -2.459453000000000000e+06 -1.716708000000000000e+06 5.301640000000000000e+05 2.922550000000000000e+05 2.173440000000000000e+06 4.750830000000000000e+05 4.716258000000000000e+06 -2.453494000000000000e+06 -5.195530000000000000e+05 3.633523000000000000e+06 -5.820080000000000000e+05 1.606849000000000000e+06 -1.156108000000000000e+06 3.214820000000000000e+05 7.915231000000000000e+06 3.676990000000000000e+05 -1.758614000000000000e+06 1.153213000000000000e+06
-1.407522000000000000e+06 1.147037000000000000e+06 1.556443000000000000e+06 1.000362000000000000e+06 -2.160419000000000000e+06 -1.949800000000000000e+04 1.446660000000000000e+05 -2.559738000000000000e+06 -1.639884000000000000e+06 1.006045000000000000e+06 -1.690151000000000000e+06 -6.148000000000000000e+03 -2.393153000000000000e+06 -4.647130000000000000e+05 1.665451000000000000e+06 -3.244160000000000000e+05 3.676990000000000000e+05 6.061325000000000000e+06 1.754370000000000000e+05 -5.399400000000000000e+04
7.776760000000000000e+05 -4.457580000000000000e+05 3.273437000000000000e+06 2.785550000000000000e+05 -2.069102000000000000e+06 1.019273000000000000e+06 -5.805070000000000000e+05 -1.913718000000000000e+06 -4.810200000000000000e+05 4.334740000000000000e+05 1.079558000000000000e+06 -1.697396000000000000e+06 -1.509820000000000000e+05 1.873400000000000000e+04 1.056871000000000000e+06 9.737690000000000000e+05 -1.758614000000000000e+06 1.754370000000000000e+05 7.434372000000000000e+06 -7.211870000000000000e+05
1.423282000000000000e+06 2.898715000000000000e+06 -1.639450000000000000e+05 -3.507103000000000000e+06 1.624883000000000000e+06 -1.483049000000000000e+06 1.575534000000000000e+06 9.640860000000000000e+05 9.185230000000000000e+05 -5.609220000000000000e+05 -1.405131000000000000e+06 5.564200000000000000e+05 -1.858485000000000000e+06 -1.022423000000000000e+06 1.263718000000000000e+06 -1.173720000000000000e+05 1.153213000000000000e+06 -5.399400000000000000e+04 -7.211870000000000000e+05 5.602639000000000000e+06
