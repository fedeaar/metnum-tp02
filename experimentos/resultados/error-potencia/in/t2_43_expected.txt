2.406356747331999987e+07
2.066793663735492900e+07
1.607344364160316065e+07
1.430376929455231316e+07
1.069542265736061148e+07
1.017505055567768402e+07
8.483526764940636232e+06
7.847579272100491449e+06
7.233398275114936754e+06
5.790052063087990507e+06
4.133710289956088644e+06
3.539035504883621354e+06
2.174081001477015670e+06
1.719427820013321005e+06
1.704858643434911501e+06
1.139631975886207307e+06
5.902220241282526404e+05
4.417216657955375849e+05
1.146541373238731467e+05
3.992301988505122608e+03
